R/shiny.R

```diff
@@ -533,15 +533,15 @@ ShinySession <- R6Class(
       }
 
       if (is.function(func)) {
-        if (length(formals(func)) != 0) {
+        funcFormals <- formals(func)
+        func <- wrapFunctionLabel(func, paste0("output$", name), ..stacktraceon = TRUE)
```

Comment:
Just checking - is the intent here to wrap the original function, and not the one created in the `if` statement below?